pkg/proxy/ipvs/ipset.go

```diff
@@ -78,6 +78,9 @@ const (
 
 	kubeHealthCheckNodePortSetComment = "Kubernetes health check node port"
 	kubeHealthCheckNodePortSet        = "KUBE-HEALTH-CHECK-NODE-PORT"
+
+	kubeIpvsSetComment = "Addresses on the ipvs interface"
```

Comment:
"`IPVS`" not "`Ipvs`", here and in all the other const/variable names. (Acronyms in golang/kubernetes names never get titled-cased; they're always either all-uppercase or all-lowercase.)